src/Rules/StarterWeb/AI/IndividualAuth/project.json

```diff
@@ -19,11 +19,15 @@
     "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0-*",$endif$
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
     "Microsoft.AspNetCore.StaticFiles": "1.0.0-*",
-    "Microsoft.EntityFrameworkCore.SqlServer": "1.0.0-*",
-    "Microsoft.EntityFrameworkCore.Tools": {
+    "Microsoft.EntityFrameworkCore.Design": {
       "version": "__ToolsPackageVersion__",
       "type": "build"
     },
+    "Microsoft.EntityFrameworkCore.SqlServer": "1.0.0-*",
+    "Microsoft.EntityFrameworkCore.SqlServer.Design": {
+      "version": "1.0.0-*",
```

Comment:
Use __AspNetCorePackageVersion__ here instead of 1.0.0-* (only for the SqlServer.Design package.